src/common/schemas/input/sign.json

```diff
@@ -10,7 +10,22 @@
     "secret": {
       "type": "string",
       "format": "secret",
-      "description": "The secret of the account that is initiating the transaction."
+      "description": "The secret of the account that is initiating the transaction.(This field is exclusive with keypair)."
+    },
+    "keypair": {
+      "type": "object",
+      "properties": {
+        "privateKey": {
+          "type": "string",
+          "description": "The uppercase hexadecimal representation of the secp256k1 or Ed25519 private key of the account that is initiating the transaction."
```

Comment:
I think this would be fine without `of the account that is initiating the transaction`
Same with `publicKey` below